samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties

```diff
@@ -388,7 +388,7 @@ Konfigurazioa aldatu da azterketa argitaratu denean.
 seb_exam_keys_recommended=Azterketara sartzeko pasahitza jartzea gomendagarria da, Safe Exam Brower-en konfigurazioa agian ez baita automatikoki balidatzen.
 
 # S2U-5
-associate_label_dyn=Sortu errubrika dinamikoa galdera hau kalifikatzeko
+associate_label_dyn=Sortu errubrika dinamikoa galdera hau kalifikatzeko (oculta siempre al estudiante durante el examen)
```

Comment:
@pablosanroman please, can you complete this text? Thanks as always!

Reply:
@PabloSanRoman a little reminder in case you've missed it. Thanks!

Reply:
Hi @bgarciaentornos , thank you, indeed, I had missed it. 
associate_label_dyn=Sortu errubrika dinamikoa galdera hau kalifikatzeko (ezkututa dago beti ikaslearentzat)